phpbench.json.dist

```diff
@@ -5,6 +5,7 @@
     "runner.bootstrap": "vendor/autoload.php",
     "runner.file_pattern": "*Bench.php",
     "runner.path": "benchmark",
+    "runner.php_config": { "memory_limit": "1G" },
```

Comment:
50MB file takes a lot more than expected to be downloaded in-memory.

Reply:
I need to profile why this is taking so much memory.

Reply:
For reference, it's the `substr` that uses the most memory.
https://github.com/mongodb/mongo-php-library/blob/ec6c431acc87ced5e85a163b92e76a8d8f69e000/src/GridFS/ReadableStream.php#L166-L174 
<img width="851" alt="image" src="https://github.com/mongodb/mongo-php-library/assets/400034/41f9cf3b-8a1f-4ffb-b85d-2cd9a1211421">